### PR TITLE
Link investigate issue to quarantine 

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
Why is this being unquarantined while the referenced issue is still open and was not yet investigated: https://github.com/dotnet/aspnetcore/issues/21521 ?

File: src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs [hunk 1]
```diff
@@ -100,7 +100,7 @@ public async Task GracefulShutdownWaitsForRequestsToFinish()
         }
 
         [ConditionalFact]
-        [QuarantinedTest] // Test still quarantined due to Sockets.Functional tests.
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/21521")] // Test still quarantined due to Sockets.Functional tests.
```
Comment (pranavkm) [inline src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs:103]: Still quarantined.

Comment (mkArtakMSFT) [reply src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs:103]: Oh, my bad - I somehow misread that as the line is being removed.

**Review by mkArtakMSFT — APPROVED** [review]

**Comment by Tratcher** [comment]
What pass data are you using to verify this? WhatsBroken only reports 5 days worth of data at the moment. We should hold unquaratining until we address the test history issue.

**Comment by pranavkm** [comment]
Gotcha. I was using the instructions here: https://github.com/dotnet/aspnetcore-internal/blob/master/docs/operations/unquarantine_tests.md, but didn't realize we need to pay attention to the date. I'll update the notes to call this out.

**Pull request merged**